alacritty_terminal/src/term/cell.rs

```diff
@@ -128,6 +182,28 @@ impl Cell {
     pub fn underline_color(&self) -> Option<Color> {
         self.extra.as_ref()?.underline_color
     }
+
+    /// Set hyperlink.
+    pub fn set_hyperlink(&mut self, hyperlink: Option<Hyperlink>) {
+        let drop_hyperlink = hyperlink.is_none()
+            && self.extra.as_ref().map_or(true, |extra| {
+                !extra.zerowidth.is_empty() || extra.underline_color.is_some()
+            });
```

Comment:
```suggestion
        let should_drop = hyperlink.is_none()
            && self.extra.as_ref().map_or(true, |extra| {
                !extra.zerowidth.is_empty() || extra.underline_color.is_some()
            });
```

I think that makes things a little more descriptive. Especially because we're dropping extra, not hyperlink.